src/feature_draw_bitmap.c

```diff
@@ -17,15 +17,18 @@ static void layer_update_callback(Layer *layer, GContext* ctx) {
   // We make sure the dimensions of the GRect to draw into
   // are equal to the size of the bitmap--otherwise the image
   // will automatically tile. Which might be what *you* want.
+  const uint8_t offset = PBL_IF_ROUND_ELSE(17, 0);
 
-#ifdef PBL_PLATFORM_BASALT
+#ifdef PBL_SDK_3
   GSize image_size = gbitmap_get_bounds(s_image).size;
```

Comment:
Since 3.0-beta6 you only need this one line. `gbitmap_get_bounds()` is available on all platforms now.